common/persistence/visibility/store/elasticsearch/query_interceptors_test.go

```diff
@@ -117,9 +117,9 @@ func (s *QueryInterceptorSuite) TestStatusProcessFunc() {
 		returnErr bool
 	}{
 		{value: "Completed", returnErr: false},
-		{value: "WORKFLOW_EXECUTION_STATUS_RUNNING", returnErr: false},
+		{value: "Running", returnErr: false},
 		{value: "1", returnErr: false},
-		{value: "", returnErr: false},
```

Comment:
What did we do previously in this case? How did this trickle down to ES?

Reply:
It queries `ExecutionStatus=''` which would return empty result.